src/components/org/OrgAdmin/common/Modal/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { IconInfoCircle } from '@sopt-makers/icons';
+import { MouseEvent } from 'react';
+
+import {
+  StInfoCloseButton,
+  StInfoDescription,
+  StInfoImg,
+  StInfoSubDescription,
+  StInfoTitle,
+  StInfoWrapper,
+} from './style';
+
+interface ModalProps {
+  title: string;
+  description: string;
+  subDescription: string;
+  imgSrc: string;
+  isInfoVisible: boolean;
+  onInfoToggle: (e: MouseEvent<HTMLButtonElement>) => void;
+}
+
+const Modal = ({
+  title,
+  description,
+  subDescription,
+  imgSrc,
+  isInfoVisible,
+  onInfoToggle,
+}: ModalProps) => {
+  return (
+    <StInfoWrapper isVisible={isInfoVisible}>
```

Comment:
```suggestion
<StInfoWrapper
  isVisible={isInfoVisible}
  aria-hidden={!isInfoVisible}
>
```

모달이 보이질 않을 때 hidden 속성을 주어 스크린 리더에 읽히지 않게 하면 좋을 거 같아요~

Reply:
좋습니다!